ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -21,6 +21,26 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: ocp-qe-perfscale-ci-tests-rosa-network-smoke
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
+    test:
+    - ref: openshift-qe-network-smoke
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-network-perf
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
+    test:
+    - ref: openshift-qe-network-perf
```

Comment:
It might be good to use a chain here and I think we can do both smoke and network perf tests together. Thoughts?

Reply:
So, is running larger network perf contingent upon smoke network perf test result?

Reply:
> I think you've said before the smoke test is a good way to see if we should continue with the larger network perf test correct?

Right - it is just to make sure the workload can property execute, prior to attempting to run a larger set of tests with more samples.

> So, is running larger network perf contingent upon smoke network perf test result?
Yeah, think of it as a gate.

Reply:
Joe, one option we could try is:

            test:
                - as: ocp-qe-perfscale-ci-tests-rosa-network-perf
                  best_effort: false
                  chain: openshift-qe-network-perf

Reply:
I thought `best-effort` could be used, but looks like that's for flagging test fail when step fails.

Reply:
Ah.. looks like best would be to create a new workflow for network-perf test and instead of calling rosa-sts-aws workflow, add pre and post steps in your own workflow.